### ISSUE TITLE
All videos and music start playing together at same time.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1364]
**Actual Result:**
All videos and music start playing together at same time.

**Expected Result:**
Videos and music should not start playing together at same time.

**Repro Steps:**
1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Navigate to 'Type your message' text box present in bottom of screen, type 'help' and press enter key.
3. Navigate to button of "Multiple media attachments" and activate it.
4. All multimedia will appear on the bottom of the screen.
5. Observe if all multimedia start playing together at same time or not.

**User Impact:**
If all videos and music start playing together at same time then the user who is accessing the page will not able to understand what information the attachment is providing.

**Test Environment:**
OS: Windows 10
OS Build: 17763.107
OS Version: 1809
Browser: Edge

**Attachment:**
[Usable_Multimedia.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2588129/Usable_Multimedia.zip)

**Comment by compulim** [comment]
- [ ] Set `autoPlay` to false in MockBot
- [ ] Add documentations to our media cards to hint the dev that `autoPlay` multiple media is not a good UX
   - [ ] We cannot stop animation from `autoPlay` because it is a GIF

**Comment by corinagum** [comment]
This is a bot-specific issue, not related to Web Chat implementation. Set `autoPlay` to false to resolve.
